test/simplifyExpression/basicsSearch/simplifyDoubleUnaryMinus.test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const simplifyDoubleUnaryMinus = require('../../../lib/simplifyExpression/basicsSearch/simplifyDoubleUnaryMinus');
+const testSimplify = require('./testSimplify');
+
+
+describe('simplifyDoubleUnaryMinus', function() {
+  var tests = [
+      ['--5', '5'],
```

Comment:
could you make this just 2 spaces? and same in `testSimplify.js`

I also added something to the linter issue #44 to check for this so these spacing comments don't have to be the focus for future PRs ^_^